content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -274,34 +274,37 @@ once bound to a node, a Pod will never be rebound to another node.
 
 ## Pod lifetime
 
-In general, Pods remain until a human or controller process explicitly removes them.
+In general, Pods remain until a human or
+{{< glossary_tooltip term_id="controller" text="controller" >}} process
+explicitly removes them.
 The control plane cleans up terminated Pods (with a phase of `Succeeded` or 
-`Failed`), when the number of Pods exceeds the configured threshold 
+`Failed`), when the number of Pods exceeds the configured threshold
 (determined by `terminated-pod-gc-threshold` in the kube-controller-manager).
 This avoids a resource leak as Pods are created and terminated over time.
 
-Three types of controllers are available:
+There are different kinds of resources for creating Pods:
 
-- Use a [Job](/docs/concepts/jobs/run-to-completion-finite-workloads/) for Pods that are expected to terminate,
+- Use a {{< glossary_tooltip term_id="deployment" >}},
+  {{< glossary_tooltip term_id="replica-set" >}} or {{< glossary_tooltip term_id="statefulset" >}}
+  for Pods that are not expected to terminate, for example, web servers.
+
+- Use a {{< glossary_tooltip term_id="job" >}}
+  for Pods that are expected to terminate once their work is complete;
   for example, batch computations. Jobs are appropriate only for Pods with
   `restartPolicy` equal to OnFailure or Never.
 
-- Use a [ReplicationController](/docs/concepts/workloads/controllers/replicationcontroller/),
-  [ReplicaSet](/docs/concepts/workloads/controllers/replicaset/), or
-  [Deployment](/docs/concepts/workloads/controllers/deployment/)
-  for Pods that are not expected to terminate, for example, web servers.
-  ReplicationControllers are appropriate only for Pods with a `restartPolicy` of
-  Always.
-
-- Use a [DaemonSet](/docs/concepts/workloads/controllers/daemonset/) for Pods that need to run one per
-  machine, because they provide a machine-specific system service.
+- Use a {{< glossary_tooltip term_id="daemonset" >}}
+  for Pods that need to run one per computer in your cluster, because they
+  provide a machine-specific system service.
```

Comment:
nit: need to run [what]? per computer in your cluster

Reply:
s/computer/eligible node/

Reply:
I can revise this to:
“_Use a DaemonSet for Pods that need to run one per eligible node, because they provide a machine-specific system service_” which is fine in UK English. I'm not sure how that looks in idiomatic US English; is it:
“_Use a DaemonSet for Pods that need to run as Pod one per eligible node, because they provide a machine-specific system service_” or something else?

Reply:
“Use a DaemonSet for Pods that need to run as *one pod per eligible node*, because they provide a machine-specific system service” or something else?

Reply:
For a pod that provides a machine-specific purpose, it may be desirable to have one copy of the pod running on each eligible node.  In this case, use a DaemonSet to configure the pod to run on the eligible nodes.